hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/util/StreamerUtil.java

```diff
@@ -516,7 +517,7 @@ public static boolean fileExists(FileSystem fs, Path path) {
   public static boolean isWriteCommit(HoodieTableType tableType, HoodieInstant instant, HoodieTimeline timeline) {
     return tableType == HoodieTableType.MERGE_ON_READ
         ? !instant.getAction().equals(HoodieTimeline.COMMIT_ACTION) // not a compaction
-        : !ClusteringUtil.isClusteringInstant(instant, timeline);   // not a clustering
+        : !ClusteringUtils.isClusteringInstant(instant, timeline);   // not a clustering
```

Comment:
let's merge the methods of `ClusteringUtil` in hudi-flink to `ClusteringUtils` in hudi-common. Alternatively, rename `ClusteringUtil` to `FlinkClusteringUtil`

Reply:
Will do it in another PR.